libraries/AP_GPS/AP_GPS_ERB.cpp

```diff
@@ -189,6 +189,20 @@ AP_GPS_ERB::_parse_gps(void)
             state.time_week_ms    = _buffer.stat.time;
             state.time_week       = _buffer.stat.week;
         }
+        if (   _buffer.ver.ver_high > 0
+           ||  _buffer.ver.ver_medium >= 2) {
+            state.rtk_baseline_coords_type = 1;                   // baseline uses NED coordinates
```

Comment:
No hold up on merging this, but it would be great if we had this as a MAVLink enum instead of having to read the comments for the values 0/1.